pkg/operator/ceph/config/keyring/admin.go

```diff
@@ -49,6 +49,6 @@ func (s *SecretStore) Admin() *AdminStore {
 
 // CreateOrUpdate creates or updates the admin keyring secret with cluster information.
 func (a *AdminStore) CreateOrUpdate(c *cephconfig.ClusterInfo) error {
-	keyring := fmt.Sprintf(adminKeyringTemplate, c.AdminSecret)
+	keyring := fmt.Sprintf(adminKeyringTemplate, c.CephCred.Secret)
```

Comment:
It feels strange to have the generic `c.CephCred.Secret` which is calling `adminKeyringTemplate`.

Reply:
I'm going to take another look to see if I can get rid of the `AdminStore` type. I believe whether or not it is the admin secret doesn't even matter for this helper.